app/facades/food_facade.rb

```diff
@@ -0,0 +1,8 @@
+class FoodFacade 
+    def self.food_finder(food)
+        food = FoodService.search_food(food)
+        food[:foods].map do |food_data|
+            Food.new(food_data, food[:totalHits])
```

Comment:
The totalHIts key is outside the foods array (it's a top level JSON key)

Keep in mind that Ruby allows you to return more than one object from a method, so you could consider returning both your Foods array and food[:totalHits]

I appreciate that you did try to do this in just one API call though -- nice work trying to minimize your external calls to Food API!